src/css/panel.base.css

```diff
@@ -5,37 +5,48 @@
 
 .progress-bar {
   position: relative;
-  height: 34px;
+  height: 32px;
+  margin-bottom: 8px;
 }
 
-.progress-bar-active {
-  font-weight: bold;
+.progress-bar .progress-bar-line-row {
+  display: flex;
+  justify-content: flex-start;
+  margin-top: 0px;
+  width: calc(100% - 8px);
+  border-radius: 2px;
 }
 
-.progress-bar .list-item-content {
-  display: block;
-  width: 99%;
+.progress-bar .bars-container {
+  display: flex;
 }
 
-.progress-bar .list-item-content .list-item-value {
-  position: absolute;
-  z-index: 3;
-  right: 0;
+.progress-bar .progress-bar-value {
+  margin-right: 14px;
+  margin-bottom: 0px;
 }
 
-.progress-bar .list-item-content .progress-bar-line {
+.progress-bar .progress-bar-line-row .progress-bar-line {
   z-index: 2;
   top: 0px;
   background: darkgreen;
-  border-right-width: 2px;
-  border-right-style: solid;
-  border-right-color: gray;
+  border-radius: 2px;
+}
+
+.progress-bar .progress-bar-title-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 4;
+  width: 100%;
+  height: 16px;
 }
 
 .progress-bar .progress-bar-title {
-  margin-bottom: -8px;
+  margin-bottom: 0px;
+  margin-left: 4px;
 }
 
 .options-tab .section {
-  margin-right: 1rem
+  margin-right: 1rem;
```

Comment:
I afraid of using `px` together with `rem`, but I hope you guys know what you are doing here